test/user.test.js

```diff
@@ -401,6 +401,29 @@ describe('User', function() {
     var pass73Char = pass72Char + '3';
     var passTooLong = pass72Char + 'WXYZ1234';
 
+    it('rejects empty passwords creation', function(done) {
+      try {
+        User.create({email: 'b@c.com', password: ''}, function(err) {
+          if (err) return done(err);
+          done(new Error('User.create() should have thrown an error.'));
+        });
+      } catch (e) {
```

Comment:
> since the `try/catch` was used for the rest of the test cases for the same topic

In my understanding, this `try/catch` approach is actually documenting the bug reported in https://github.com/strongloop/loopback/issues/1364. We need to fix the code so that password-validation errors are reported via callback, so that `try/catch` based tests in this file can be reworked to the usual callback-based style.